code/modules/mob/living/carbon/xenomorph/abilities/ravager/ravager_abilities.dm

```diff
@@ -65,11 +65,11 @@
 	action_type = XENO_ACTION_CLICK
 	ability_primacy = XENO_PRIMARY_ACTION_1
 	plasma_cost = 0
-	xeno_cooldown = 17 SECONDS
+	xeno_cooldown = 16 SECONDS
```

Comment:
keep the cooldown at 17 (i.e. increase by 1 second)

Reply:
xir yes xir